blog.go

```diff
@@ -0,0 +1,41 @@
+package shopify
+
+import (
+	"time"
+)
+
+// Blogs is a list of blog
+type Blogs []Blog
+
+// Blogs is a shopify blog
+type Blog struct {
+	// Commentable indicates whether readers can post comments to the blog and if comments are moderated or no
+	Commentable string
+	// CreatedAt The date and time (ISO 8601 format) when the article was created.
+	CreatedAt time.Time
+	// Handle is a unique, human-readable string for the collection automatically generated from its title.
+	/* This is used in themes by the Liquid templating language to refer to the collection. (limit: 255 characters) */
```

Comment:
```suggestion
	/* This is used in themes by the Liquid templating language to refer to the blog. (limit: 255 characters) */
```

Reply:
Should not be referring to liquid templating in this package. It should not know anything about how it is intended to be used.

Reply:
Agree actually